packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -113,3 +116,163 @@ class DefaultTaggedText extends StatelessWidget {
     );
   }
 }
+
+/// A widget that takes two children and lays them out along one axis.
+///
+/// The user can customize the amount of space allocated to each child by
+/// dragging a divider between them.
+class Split extends StatefulWidget {
+  /// Builds a [Split] with [Axis.horizontal] direction.
+  ///
+  /// [firstChild] will be placed before [secondChild] in a [Row].
+  const Split.horizontal({
+    Key key,
+    @required Widget firstChild,
```

Comment:
instead of firstChild and second child, why not use `children:`
If you only support two children you can always assert that there are exactly two children.

Reply:
`firstChild` and `secondChild` explain exactly what the code does without a need for asserts.

It's simpler, and this pattern has [prior art](https://api.flutter.dev/flutter/widgets/AnimatedCrossFade/firstChild.html).

Unless we plan on supporting n != 2 children, I'd like to leave this as-is.

Reply:
my guess is at some point we will want to support n > 2 children.